docs/source/pages/overview.rst

```diff
@@ -109,6 +109,29 @@ the native `MetricCollection`_ module can also be used to wrap multiple metrics.
             val3 = self.metric3['accuracy'](preds, target)
             val4 = self.metric4(preds, target)
 
+Metrics in Dataparallel (DP) mode
+=================================
+
+When using metrics in `Dataparallel (DP) <https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html#torch.nn.DataParallel>`_
+mode, one should be aware DP will both create and destroy replicates of ones metric during a single forward pass. 
```

Comment:
maybe not "destroy", rather clean-up or delete?

Reply:
I agree with @Borda . "DP will both create and clean-up replicas of Metric objects"